pages/projusers/index.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: page
+title: Who uses Scikit-HEP?
+permalink: /projusers
+nav_order: 50
+has_children: true
+---
+
+Who uses Scikit-HEP? {#scikit_hep_users}
+====================
+
+This page gives an overview of who or what is using one or more packages
+of the Scikit-HEP project:
+
+- High Energy Physics experiments or collaborations.
```

Comment:
These could link to the pages below too. We do something similar in developer.

Reply:
I could simply add
```
[projects]: {{ site.baseurl }}{% link pages/projusers/projects.md %}
[publications]: {{ site.baseurl }}{% link pages/projusers/publications.md %}
```
after this list, since the list (4 items) is a bit more explicit than the pages to be linked (2).
Is this what you mean?

Reply:
Yes, that's what I'm referring too. This is sort of an "index" page.

Reply:
Added.